core/src/main/java/com/digitalpebble/stormcrawler/persistence/DefaultScheduler.java

```diff
@@ -17,12 +17,7 @@
 import com.digitalpebble.stormcrawler.Constants;
 import com.digitalpebble.stormcrawler.Metadata;
 import com.digitalpebble.stormcrawler.util.ConfUtils;
-import java.util.Calendar;
-import java.util.Date;
-import java.util.HashMap;
-import java.util.Iterator;
-import java.util.Map;
-import java.util.Optional;
+import java.util.*;
```

Comment:
Start import intended?

Reply:
It was not corrected by the formatter. so i think thats ok.